.mci.yml

```diff
@@ -98,6 +109,9 @@ buildvariants:
       display_name: "Ubuntu 14.04 Release"
       expansions:
           build_type: "Release"
+          source: *linux_source
```

Comment:
since we have an open ticket to add a RHEL builder, can you change these to `ubuntu_source`, `ubuntu_tar_options` etc?